~

```diff
@@ -0,0 +1,29 @@
+pick c321fe670 [send] Make send RPCs return fee reason
```

Comment:
uhhh, this looks like an accidental file addition? :)

Reply:
Fixed it